pkg/cli/commands.go

```diff
@@ -1,8 +1,6 @@
 package cli
 
 import (
-	"bytes"
-	"encoding/binary"
```

Comment:
nitpick: stringToInt64, intToScalar, stringToScalar, generateDualKey in cli pkg seem are unused for now.

Reply:
Correct - removing them.